management-api-server/src/main/java/com/datastax/mgmtapi/resources/models/Table.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Please see the included license file for details.
+ */
+package com.datastax.mgmtapi.resources.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Map;
+
+public class Table {
+  @JsonProperty(value = "name", required = true)
+  public final String name;
+
+  @JsonProperty(value = "compaction")
+  public final Map<String, String> compaction;
+
+  @JsonCreator
+  public Table(
+      @JsonProperty("name") String name,
+      @JsonProperty("compaction") Map<String, String> compaction) {
+    this.name = name;
+    this.compaction = compaction;
+  }
```

Comment:
suggestion:
I know it's not necessary, but maybe implement `hashCode()`, `equals()` and `toString()`? I'm fine without those if you don't see any real value.